_docs/how-to/python-package.md

```diff
@@ -0,0 +1,240 @@
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  build-n-publish:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: astral-sh/setup-uv@v5
+      - run: uv build
+      - uses: pypa/gh-action-pypi-publish@release/v1
+```
+
+Make sure that your repository only allows trusted contributors write access.
+The team manager and another truster team member should be the package owners on PyPI,
+using their Canonical email addresses.
+Make sure to also claim your package on [Test PyPI](https://test.pypi.org/),
+and setup a workflow for publishing there.
+All team members can be owners on Test PyPI.
+
+A major benefit of publishing on PyPI is that users of your library can specify version ranges in their dependencies.
+Therefore, if you’re going to publish on PyPI, we highly recommend that you use semantic versioning for your library.
+
+A non-dev/alpha/beta/etc qualified 1.x release to PyPI signifies that your library is ready for public consumption.
+You should also communicate this through the ["Development Status" Trove classifier](https://pypi.org/classifiers/) in your `pyproject.toml`.
+
+
+(python-package-distribution-git)=
+### Git
+
+You can get started by distributing your library as a Python package with very little friction using GitHub.
+This is good for prototyping, or when first transitioning from a charmcraft-style library to a Python package,
+and may be a good fit for libraries that are intended for team-internal use.
+If your library is intended for external users, consider whether PyPI would be a better choice.
+
+You’ll need to include `git` in your charm’s build dependencies to use a GitHub-hosted library in your charm:
+
+```yaml
+parts:
+  charm:
+    build-packages: [git]
+```
+
+Then you can specify the dependency in your requirements like this:
+
+```
+charmlibs-pathops @ git+https://github.com/canonical/charmtech-charmlibs@main#subdirectory=pathops
+```
+
+You can specify any branch, tag, or commit after the `@`.
+If you leave it off, it will default to `@main`.
+You can’t specify a version range.
+This can make dependency resolution problematic for users,
+especially if your library is depended on by other charm libraries.
+Tools that scan versions for security vulnerabilities may also struggle with such dependencies.
+
+If your package is in a subdirectory of your repository,
+for example in a monorepo (like `charmlibs-pathops`),
+or when developing libraries alongside charms,
+you'll need to specify the subdirectory.
+If your library has a dedicated repository,
+leave off the subdirectory and it will default to the repository root.
+
+In `pyproject.toml`, quote the entire string above in your dependencies list,
+or use `uv add git+...` to have `uv` add `charmlibs-pathops` to your dependencies list,
+and the git referece to `tool.uv.sources`.
+For `poetry` see [here](https://python-poetry.org/docs/dependency-specification/#git-dependencies).
```

Comment:
It's best to avoid using words like "here" as link anchors (for accessibility for example).